src/main/kotlin/com/terraformation/backend/support/atlassian/resource/CreateServiceRequest.kt

```diff
@@ -0,0 +1,57 @@
+package com.terraformation.backend.support.atlassian.resource
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.terraformation.backend.support.atlassian.model.ServiceRequestFieldsModel
+import io.ktor.client.call.body
+import io.ktor.client.request.HttpRequestBuilder
+import io.ktor.client.request.setBody
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.HttpMethod
+import io.ktor.http.path
+
+/**
+ * The POST request to create a service ticket.
+ *
+ * @param requestFieldValues the title of the Jira issue
+ * @param requestTypeId the ID of the Jira issue type
+ * @param reporter the email address of the reporter
+ * @param serviceDeskId the ID for the service desk
+ */
+class CreateServiceDeskRequest(
```

Comment:
Minor nit: Filename doesn't match this class name. (Technically not required, of course, but the names are close enough that it seems like they were intended to be the same.)